balancer-js/src/types.ts

```diff
@@ -46,16 +61,32 @@ export interface BalancerNetworkConfig {
       lbpRaisingTokens?: string[];
       stETH?: string;
       wstETH?: string;
+      bal: string;
+      veBal: string;
+      bbaUsd: string;
     };
   };
   urls: {
     subgraph: string;
+    gaugesSubgraph: string;
+    blockNumberSubgraph?: string;
   };
   pools: {
     wETHwstETH?: PoolReference;
   };
 }
 
+export interface BalancerDataRepositories {
+  pools: Findable<Pool, PoolAttribute> & Searchable<Pool> & Updatetable<Pool>;
```

Comment:
I don't think pools should *have* to implement updatable as users should be able to use read-only sources.

Reply:
Also should data repositories even be writeable? Seems like we should have DataWriters or something instead. I think of repositories as a place you fetch data from, not use for updating data.